src/services/moderation.service.ts

```diff
@@ -331,6 +318,110 @@ export class ModService {
     return reply;
   }
 
+  public async getRullReport(guild: Guild, user_handle: string) {
+    const collections = await this._storageService.getCollections();
+    const id = await Moderation.Helpers.resolveUser(guild, user_handle);
+    if (!id) {
+      throw new Error('User not found');
+    }
+
+    const modreports = collections?.modreports;
+    const modwarnings = collections?.modwarnings;
+
+    const reports = await modreports?.find({ guild: guild.id, user: id }).toArray();
+    const warnings = await modwarnings?.find({ guild: guild.id, user: id }).toArray();
+    const banStatus = await this._getBanStatus(collections, guild, id);
+
+    if (!reports) {
+      throw new Error('Couldnt get reports');
+    }
+
+    //Number of Reports > warns
```

Comment:
`//<space>`

Reply:
does `prettier` not do that for u

Reply:
I dont think so :(, lemme see if there's a setting